src/main/kotlin/org/rust/cargo/runconfig/wasmpack/WasmPackCommandConfigurationType.kt

```diff
@@ -0,0 +1,38 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust.cargo.runconfig.wasmpack
+
+import com.intellij.execution.configurations.ConfigurationFactory
+import com.intellij.execution.configurations.ConfigurationTypeBase
+import com.intellij.execution.configurations.ConfigurationTypeUtil
```

Comment:
Please optimize imports. It can be done right from [commit dialog](https://www.jetbrains.com/help/idea/commit-changes-dialog.html#before_commit)